packages/grid/_modules_/grid/components/containers/GridRootStyles.ts

```diff
@@ -230,6 +230,17 @@ export const useStyles = makeStyles(
           whiteSpace: 'nowrap',
           borderBottom: `1px solid ${borderColor}`,
         },
+        '& .MuiDataGrid-editCellInputBase': {
+          ...theme.typography.body2,
+          border: `1px solid ${borderColor}`,
+          borderRadius: theme.shape.borderRadius,
+          '& :focus': {
+            borderColor: theme.palette.text.primary,
+            outline: 0,
+            // boxShadow: getThemePaletteMode(theme.palette) === 'light' ? null : '0 0 0 100px #266798 inset',
+            boxShadow: 'inset 0 1px 1px rgb(0 0 0 / 8%), 0 0 8px rgb(102 175 233 / 60%)',
```

Comment:
not sure about what we want here...

Reply:
Not a huge fan of this style. Happy to give it a try if you are not happy with it either.

Reply:
Please do. If that's not blocking, I suggest we merge this one and open new ones for the rest of the changes

Reply:
Definitely not blocking